### PR TITLE
Suggestion names in component colors

### LINKED ISSUE #1393: Make color names more generic to avoid confusion with Aletheia pallete
Question: @thesocialdev I wonder if we could use more generic names, the color prefix could confuse users if their color pallete is based on orange for example

_Originally posted by @pepermao in https://github.com/AletheiaFact/aletheia/pull/1384#discussion_r1790466611_

### PR DESCRIPTION
**Pull request by LuizFNJ**
OBS: I did a rebase and I already took advantage of this PR to update the configuration environment with the new names too

# Description
*This PR will be to define more generic names in colors. In this first commit my intention is to receive feedback on the names, defining the names the next commit will be to change the names in hardCoded*

*Example suggestion:*
- Colors.blue   ⭢   Colors.Primary
- Colors.gray   ⭢   Colors.Neutral
- Colors.redText   ⭢   Colors.error

Fixes #1393 

# Type of change
*Please delete options that are not relevant.*

- [ ] Bug fix (non-breaking change which fixes an issue)     
- [ ] New feature (non-breaking change which adds functionality)     
- [x] Existing feature enhancement (non-breaking change which modifies existing functionality)     

# Testing
*Provide relevant testing instructions. What scenarios are impacted? What build may be necessary to test this change?*

# Developer Checklist
## General
- [x] Code is appropriately commented, particularly in hard-to-understand areas     
- [x] Repository documentation has been updated (Readme.md) with additional steps required for a local environment setup.
- [x] No `console.log` or related logging is added.
- [x] No code is repeated/duplicated in violation of DRY. The exception to this is for new (MVP/Prototype) functionality where the abstraction layer may not be clear (comments should be added to explain the violation of DRY in these scenarios).   
- [x] Documented with TSDoc all library and controller new functions

## Frontend Changes
- [x] No new styling is added through CSS files (Unless it's a bugfix/hotfix)
- [x] All types are added correctly

### Tests
- [x] All existing unit and end to end tests pass across all services     
- [ ] Unit and end to end tests have been added to ensure backend APIs behave as expected     

# Merge Request Review Checklist 
- [ ] An issue is linked to this PR and these changes meet the requirements outlined in the linked issue(s)     
- [ ] High risk and core workflows have been tested and verified in a local environment.     
- [ ] Enhancements or opportunities to improve performance, stability, security or code readability have been noted and documented in JIRA issues if not being addressed.     
- [ ] Any dependent changes have been merged and published in downstream modules     
- [ ] Changes to multiple services can be deployed in parallel and independently. If not, changes should be broken out into separate merge requests and deployed in order.

### CODE REVIEW

**Review by pepermao — CHANGES_REQUESTED** [review]
issue: you have to update the color name on each component as well

**Comment by LuizFNJ** [comment]
> problema: você tem que atualizar o nome da cor em cada componente também

**Review by pepermao — APPROVED** [review]
LGTM: Another set of eyes on the suggestion names would be great

File: src/styles/colors.ts [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+    neutral: "rgb(76, 77, 78)", //#4d4e4f
+    neutralSecondary: "rgb(151, 154, 155)", //#979a9b
+    neutralTertiary: "rgb(194, 200, 204)", //#c2c8cc
+    lightNeutral: "rgb(245, 245, 245)", //#f5f5f5
+    lightNeutralSecondary: "rgb(238, 238, 238)", //#eeeeee
+    white: "rgb(255, 255, 255)", //#ffffff
     warning: "rgba(219, 159, 13, 0.3)", //#db9f0d
-    logoWhite: "#E8E8E8",
-    redText: "#ff4d4f",
+    Shadow: "rgba(0, 0, 0, 0.25)",
```
Comment (thesocialdev) [inline src/styles/colors.ts:19]: nitpick: shadow is with capital letter and we should be using camel-case convention.

Comment (thesocialdev) [reply src/styles/colors.ts:19]: question: how will that affect the localConfig schema?

**Review by thesocialdev — CHANGES_REQUESTED** [review]

File: src/styles/colors.ts (hunk 1 above)
Comment (LuizFNJ) [reply src/styles/colors.ts:19]: I believe that it will only be necessary to correct the localConfig schema when the stage emerges, correct?

**Review by pepermao — APPROVED** [review]

**Review by thesocialdev — APPROVED** [review]

**Pull request merged**